torch/_refs/__init__.py

```diff
@@ -1565,6 +1573,15 @@ def cat(tensors: TensorSequenceType, dim: int = 0) -> TensorLikeType:
     return prims.cat(filtered, dim)
 
 
+@out_wrapper
+def column_stack(tensors: TensorSequenceType) -> TensorLikeType:
+    aligned_tensors = [
+        x if x.ndim > 1 else torch._prims.expand_dims(x, list(range(x.ndim, 2)))
+        for x in tensors
+    ]
+    return prims.cat(aligned_tensors, 1)
```

Comment:
Prefer using the reference to the prim (so just `cat(...)`)

Reply:
Any reason that we prefer to use references inside _refs instead of prim? Using `prims` in references looks like a cleaner implementation.